src/plugins/dashboard/public/application/lib/save_dashboard.ts

```diff
@@ -54,7 +55,9 @@ export function saveDashboard(
   // TODO: should update Dashboard class in the if(id) block
```

Comment:
Is this TODO still necessary?

Reply:
I think it's worth it to keep as we as a fast follow we should look into using redux slice to keep track of a dashboard and should be updated in this block.